rules/do-you-use-co-creation-patterns/rule.md

```diff
@@ -60,14 +60,14 @@ We can avoid this by using co-creation patterns.
 
 > As a general rule pull requests with less that 20 lines of code and larger changes with a degree of complexity or risk make good candidates for co-creation
 
-### Techniques
+### Patterns
```

Comment:
Is the way we work a pattern?
I would think this is more like a best practice or something like that.

Reply:
I changed due to the rule being called 'co-creation patterns' yet, I hadn't referenced any patterns anywhere.

Reply:
I'd stick with patterns... as in "patterns of behaviour" makes sense for how can work together in different ways :)